client/rpcserver/types.go

```diff
@@ -54,43 +54,66 @@ type tradeResponse struct {
 
 // openWalletForm is information necessary to open a wallet.
 type openWalletForm struct {
-	AssetID uint32           `json:"assetID"`
-	AppPass encode.PassBytes `json:"appPass"`
+	assetID uint32
+	appPass encode.PassBytes
 }
 
 // newWalletForm is information necessary to create a new wallet.
 type newWalletForm struct {
-	AssetID    uint32           `json:"assetID"`
-	Account    string           `json:"account"`
-	ConfigText string           `json:"config"`
-	WalletPass encode.PassBytes `json:"walletPass"`
-	AppPass    encode.PassBytes `json:"appPass"`
+	assetID    uint32
+	account    string
+	configText string
+	walletPass encode.PassBytes
+	appPass    encode.PassBytes
 }
 
 // helpForm is information necessary to obtain help.
 type helpForm struct {
-	HelpWith         string `json:"helpwith"`
-	IncludePasswords bool   `json:"includepasswords"`
+	helpWith         string
+	includePasswords bool
 }
 
 // tradeForm combines the application password and the user's trade details.
 type tradeForm struct {
-	AppPass encode.PassBytes
-	SrvForm *core.TradeForm
+	appPass encode.PassBytes
+	srvForm *core.TradeForm
 }
 
 // cancelForm is information necessary to cancel a trade.
 type cancelForm struct {
-	AppPass encode.PassBytes `json:"appPass"`
-	OrderID string           `json:"orderID"`
+	appPass encode.PassBytes
+	orderID string
 }
 
 // withdrawForm is information necessary to withdraw funds.
 type withdrawForm struct {
-	AppPass encode.PassBytes `json:"appPass"`
-	AssetID uint32           `json:"assetID"`
-	Value   uint64           `json:"value"`
-	Address string           `json:"address"`
+	appPass encode.PassBytes
+	assetID uint32
+	value   uint64
+	address string
+}
+
+// orderBookForm is information necessary to fetch an order book.
+type orderBookForm struct {
+	host    string
+	base    uint32
+	quote   uint32
+	nOrders uint64
+}
+
+// MiniOrder is minimal information about an order in a market's order book.
+type MiniOrder struct {
+	Qty       float64 `json:"qty"`
+	Rate      float64 `json:"rate"`
+	Sell      bool    `json:"sell"`
+	Token     string  `json:"token"`
+	Immediate bool    `json:"immediate"`
```

Comment:
An order in an order book will always be immediate = false. Why not just use the `core.MiniOrder` type here?